lib/env.ts

```diff
@@ -40,18 +40,22 @@ const jwtSigningKeys = {
 };
 const openid = { jwsAlg, jwtSigningKeys };
 
-export default {
-  hostUrl,
-  hostPort,
+const jacksonOptions: JacksonOption = {
```

Comment:
@niwsa Removing `hostUrl` and `hostPort`, not used anywhere in the codebase.

Reply:
It is used to construct externalUrl

Reply:
But you are right, doesn't have to be exported